integration-tests/tests/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "tsc -p .",
     "start": "ts-mocha --project ./tsconfig.json --require src/utils/segfault-handler.js --require src/utils/inject-globals.js --watch-extensions ts,js --watch src/index.ts",
     "prepack": "npm run build",
-    "lint": "tslint -p ."
+    "lint": "../../node_modules/.bin/eslint . --ext .js,.ts"
```

Comment:
Could this be handles in `.eslintrc.json` through files, instead of `--ext .js,.ts` here? Then each project defines if linting should be done for additional `jsx`/`tsx` extensions.